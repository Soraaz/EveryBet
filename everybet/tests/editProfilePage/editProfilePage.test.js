import React from "react";
import renderer from "react-test-renderer";

import EditProfilePage from "../../src/editProfilePage/editProfilePage.js";

describe("EditProfilePage", () => {

	describe("rendering", () => {
		it("Should render correctly", () => {
			const tree = renderer
				.create(<EditProfilePage/>)
				.toJSON();
			expect(tree).to.matchSnapshot();
		});
	});
});
