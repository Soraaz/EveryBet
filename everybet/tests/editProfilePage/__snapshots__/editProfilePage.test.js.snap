// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditProfilePage rendering Should render correctly 1`] = `
<RCTSafeAreaView
  style={
    Array [
      Object {
        "flex": 1,
        "justifyContent": "flex-start",
      },
      Object {
        "backgroundColor": "#F4F4FE",
      },
    ]
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#37597F",
        "flexDirection": "row",
        "height": 60,
        "justifyContent": "flex-start",
      }
    }
  >
    <Image
      accessible={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      source={
        Object {
          "testUri": "../../../assets/images/everybet_logo.png",
        }
      }
      style={
        Object {
          "height": 39.25,
          "marginLeft": 25,
          "width": 138,
        }
      }
    />
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#FFFFFF",
          "borderRadius": 10,
          "flexDirection": "row",
          "height": 20,
          "justifyContent": "flex-start",
          "marginLeft": "auto",
          "width": 90,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../assets/images/betcoin_icon.png",
          }
        }
        style={
          Object {
            "height": 30,
            "width": 30,
          }
        }
      />
      <Text
        style={
          Object {
            "fontFamily": "System",
            "fontSize": 10,
            "marginLeft": "auto",
            "marginRight": 5,
            "textAlign": "right",
          }
        }
      >
        0
      </Text>
    </View>
    <Image
      accessible={true}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      source={
        Object {
          "testUri": "../../../assets/images/user_placeholder.png",
        }
      }
      style={
        Object {
          "borderRadius": 20,
          "height": 40,
          "marginHorizontal": 15,
          "width": 40,
        }
      }
    />
  </View>
</RCTSafeAreaView>
`;
